package.json

```diff
@@ -265,20 +277,97 @@
           "group": "d2"
         }
       ]
-    },
-    "themes": [
+    }
+  },
+  "menus": {
+    "commandPalette": [
       {
-        "label": "D2 Light",
-        "uiTheme": "vs",
-        "path": "./themes/light-color-theme.json"
+        "command": "D2.ShowPreviewWindow",
+        "when": "editorLangId == d2",
+        "group": "d2"
       },
       {
-        "label": "D2 Dark",
-        "uiTheme": "vs-dark",
-        "path": "./themes/dark-color-theme.json"
+        "command": "D2.CompileToSvg",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.PickTheme",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.PickLayout",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.ToggleSketch",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      }
+    ],
+    "editor/context": [
+      {
+        "command": "D2.ShowPreviewWindow",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.CompileToSvg",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      }
+    ],
+    "editor/title": [
+      {
+        "command": "D2.ShowPreviewWindow",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.CompileToSvg",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      }
+    ],
+    "editor/title/context": [
+      {
+        "command": "D2.ShowPreviewWindow",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.CompileToSvg",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      }
+    ],
+    "explorer/context": [
+      {
+        "command": "D2.ShowPreviewWindow",
+        "when": "resourceLangId == d2",
+        "group": "d2"
+      },
+      {
+        "command": "D2.CompileToSvg",
+        "when": "resourceLangId == d2",
+        "group": "d2"
       }
     ]
   },
```

Comment:
this doesn't seem right likely a merge issue, I'd just rollback to before merging and try running prettier as mentioned here:

> Just need `git checkout master -- themes` to get the json files back and (from root dir of repo) `yarn lint && yarn prettier -w .` which should also fix the package.json merge conflicts